src/helpers.js

```diff
@@ -0,0 +1,45 @@
+function getCurrencyByKey(key) {
+  if (/^eth/i.test(key))
```

Comment:
@denisfd This is very bad practice to make such amount of if cases, please avoid this.

Consider reading example below and fix your code 👍 

```js
function getCurrencyByKey(key) {
  const currencies = ['eth', 'btc', 'bch', 'ltc', 'xrp', 'dash', 'trst'];

  for (var currency in currencies) {
    if (/* condition */) { 
      return currency. toUpperCase();
  }
}
```

P.S: I can't find usage of this function, can you explain where it used?

Reply:
In deleted testcases :D